macros/Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "xtra-macros"
-version = "0.1.0"
+version = "0.6.0"
```

Comment:
Why does this need to track the version of the root crate?

Reply:
Arguably it does not, but I think this is a standard set in other crates - serde and serde_derive seem pegged, as do bevy and bevy_internal/bevy_dylib/bevy_input.

Otherwise, the option is to maintain a table like this;
![image](https://github.com/Restioson/xtra/assets/6688948/734c1960-5f5d-4129-9829-efdf689ab03b)

Reply:
I guess it does makes things easier while we are in `0.x` land and after `1.x`, the difference doesn't matter too much :)